apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/CarrierClaimTransformerV2.java

```diff
@@ -266,13 +267,24 @@ private static ExplanationOfBenefit transformClaim(
                 CcwCodebookVariable.CARR_LINE_MTUS_CNT, line.getMtusCount()));
       }
 
+      // CARR_LINE_MTUS_CNT => ExplanationOfBenefit.item.extension
+      line.getMtusCode()
+          .ifPresent(
+              code ->
+                  item.addExtension(
+                      TransformerUtilsV2.createExtensionCoding(
+                              eob, CcwCodebookVariable.CARR_LINE_MTUS_CD, code)
+                          .setUrl(
+                              CCWUtils.calculateVariableReferenceUrl(
+                                  CcwCodebookVariable.CARR_LINE_MTUS_CNT))));
+
       // CARR_LINE_MTUS_CD => ExplanationOfBenefit.item.extension
       line.getMtusCode()
           .ifPresent(
               code ->
                   item.addExtension(
                       TransformerUtilsV2.createExtensionCoding(
-                          eob, CcwCodebookVariable.CARR_LINE_MTUS_CNT, code)));
+                          eob, CcwCodebookVariable.CARR_LINE_MTUS_CD, code)));
```

Comment:
This should be unchanged for backwards compatibility

Reply:
> This should be unchanged for backwards compatibility

What I have generates the expected proposed extension according to the Jira ticket notes. You can compare my output in eobByPatientIdPaged.json line 4847 to those notes.